### PR TITLE
New Testcase: 120-Android-Hardcoded-Secrets

### PR DESCRIPTION
**Pull request by poseidontor**

### CODE REVIEW

File: vulnerabilities/120_android_hardcoded_secrets/business_implication.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Hardcoded secrets can allow a malicious actor to access private resources that may contain sensitive information leading to data breaches and the exposure of users' personal data which may lead to loss in business.
```
Comment (rajan262) [inline vulnerabilities/120_android_hardcoded_secrets/business_implication.md:2]: A new line needs to be added at the end.

File: vulnerabilities/120_android_hardcoded_secrets/description.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+Malicious actors could take advantage of the hardcoded secrets to access private databases, leading to data breaches and the exposure of users' personal data.
```
Comment (rajan262) [inline vulnerabilities/120_android_hardcoded_secrets/description.md:2]: Add new line at the end.

File: vulnerabilities/120_android_hardcoded_secrets/intro.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+This vulnerability arises because of the insecure practice to hardcode secrets in the application package, that includes API keys, server authentication credentials, etc.
```
Comment (rajan262) [inline vulnerabilities/120_android_hardcoded_secrets/intro.md:2]: Add new line at the end.

File: vulnerabilities/120_android_hardcoded_secrets/related_to.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+- [CWE-259](https://cwe.mitre.org/data/definitions/259.html)
+- [CWE-321](https://cwe.mitre.org/data/definitions/321.html)
+- [CWE-798](https://cwe.mitre.org/data/definitions/798.html)
```
Comment (rajan262) [inline vulnerabilities/120_android_hardcoded_secrets/related_to.md:4]: A new line needs to be added at the end.

File: locales/ja/120_android_hardcoded_secrets/related_to.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+- [CWE-259](https://cwe.mitre.org/data/definitions/259.html)
+- [CWE-321](https://cwe.mitre.org/data/definitions/321.html)
+- [CWE-798](https://cwe.mitre.org/data/definitions/798.html)
```
Comment (rajan262) [inline locales/ja/120_android_hardcoded_secrets/related_to.md:4]: A new line needs to be added at the end.

**Comment by poseidontor** [comment]
@rajan262 made the required changes.

**Review by rajan262 — APPROVED** [review]

**Comment by rajan262** [comment]
The force commit after approved PR was added to remove extra space at line end of some files. It was not clearly visible while doing the PR review since all files were new files.

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**